test/index.spec.ts

```diff
@@ -179,5 +192,22 @@ describe('Public API', () => {
       );
       expect(path.extname(driverPath)).toEqual('.zip');
     });
+
+    it('should download a custom version of a zip file', async () => {
+      process.env.ELECTRON_CUSTOM_VERSION = '2.0.10';
+      const zipPath = await downloadArtifact({
+        artifactName: 'electron',
+        cacheRoot,
+        downloader,
+        platform: process.env.npm_config_platform || process.platform,
+        arch: process.env.npm_config_arch || process.arch,
+        version: '2.0.3',
+      });
+      expect(typeof zipPath).toEqual('string');
+      expect(await fs.pathExists(zipPath)).toEqual(true);
+      expect(path.basename(zipPath)).toMatchInlineSnapshot(`"electron-v2.0.10-darwin-x64.zip"`);
```

Comment:
This should invoke `process.platform`, since it runs on all platforms - that's why tests are failing at present